src/stores/ErrorStore.ts

```diff
@@ -21,11 +21,7 @@ export default class ErrorStore extends Store<ErrorStoreState> {
                 lastError: action.message,
                 isDismissed: false,
             }
-        } else if (
-            action instanceof DismissLastError ||
-            action instanceof RouteRequestSuccess ||
-            (action instanceof SetCustomModelBoxEnabled && !action.enabled)
-        ) {
+        } else if (action instanceof DismissLastError || action instanceof RouteRequestSuccess) {
```

Comment:
Ah now I see what you meant. Maybe the `instanceof RouteRequestSucess` check does not really belong here either?